orchestrator/test_runner/src/erc_721_happy_path.rs

```diff
@@ -0,0 +1,638 @@
+                            amount, start_coin.denom, end_coin.amount, end_coin.denom
+                        );
+                        return Ok(());
+                    }
+                }
+            }
+            (None, Some(end_coin)) => {
+                // When a bridge governance vote happens, the orchestrator will replay all incomplete
+                // sends to cosmos on the next send to cosmos transaction, so we need to use expected_change
+                if let Some(expected) = expected_change.clone() {
+                    if end_coin.amount == expected {
+                        info!(
+                            "Successfully bridged ERC20 {}{} to Cosmos! Balance is now {}{}",
+                            amount, end_coin.denom, end_coin.amount, end_coin.denom,
+                        );
+                        return Ok(());
+                    }
+                } else if amount == end_coin.amount {
+                    info!(
+                        "Successfully bridged ERC20 {}{} to Cosmos! Balance is now {}{}",
+                        amount, end_coin.denom, end_coin.amount, end_coin.denom
+                    );
+                    return Ok(());
+                } else {
+                    panic!("Failed to bridge ERC20!")
+                }
+            }
+            _ => {}
+        }
+        info!("Waiting for ERC20 deposit");
+        contact.wait_for_next_block(TOTAL_TIMEOUT).await.unwrap();
+    }
+    Err(GravityError::InvalidBridgeStateError(
+        "Did not complete deposit!".to_string(),
+    ))
+}
+
+// Tries up to TOTAL_TIMEOUT time to find a MsgSendToCosmosClaim attestation created in the
+// test_erc20_deposit test
+async fn check_send_to_cosmos_attestation(
+    grpc_client: &mut GravityQueryClient<Channel>,
+    erc20_address: EthAddress,
+    receiver: CosmosAddress,
+    sender: EthAddress,
+) -> Result<(), GravityError> {
+    let start = Instant::now();
+    let mut found = false;
+    loop {
+        iterate_attestations(grpc_client, &mut |decoded: MsgSendToCosmosClaim| {
+            let right_contract = decoded.token_contract == erc20_address.to_string();
+            let right_destination = decoded.cosmos_receiver == receiver.to_string();
+            let right_sender = decoded.ethereum_sender == sender.to_string();
+            found = right_contract && right_destination && right_sender;
+        })
+        .await;
+        if found {
+            break;
+        } else if Instant::now() - start > TOTAL_TIMEOUT {
+            return Err(GravityError::InvalidBridgeStateError(
+                "Could not find the send_to_cosmos attestation we were looking for!".to_string(),
+            ));
+        }
+        info!("Looking for send_to_cosmos attestations");
+        delay_for(Duration::from_secs(10)).await;
+    }
+    info!("Found the expected MsgSendToCosmosClaim attestation");
+    Ok(())
+}
+
+#[allow(clippy::too_many_arguments)]
+async fn test_batch(
```

Comment:
this can be removed as there's no way to execute this functionality right now.

Reply:
Removed